### PR TITLE
feat: add Latest Media News page.

### PR DESCRIPTION
**Pull request by johnhorsema**
* **Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines
- [ ] i18n translation has been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ..., provide screenshots if necessary)

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: src/data/config/site.json [hunk 1]
```diff
@@ -5,31 +5,43 @@
...
       "title": "hygiene_tips.title",
       "to": "/hygiene-tips",
       "icon": "info",
-      "sideMenu": true
+      "sideMenu": true,
+      "bottomMenu": true
+    },
+    {
+      "title": "latest_media_news.title",
+      "to": "/latest-media-news",
```
Comment (cswbrian) [inline src/data/config/site.json]: Please change to /news

File: src/pages/hygiene-tips.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ const CardsContainer = styled(Box)`
   flex-wrap: wrap;
 `
 const CardContainer = styled(Box)`
-  flex: 0 1 calc(25% - 1em);
+  flex: 0 1 calc(33.3333% - 1em);
```
Comment (cswbrian) [inline src/pages/hygiene-tips.js]: Is this commit mixed? Looks like this is from hygiene-tips

File: src/pages/latest-media-news.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import { MediaCard } from "@components/organisms/MediaCard"
+import Box from "@material-ui/core/Box"
+
+const LatestMediaNewsPage = ({ data, pageContext }) => {
+  const { t } = useTranslation()
+  return (
+    <Layout>
+      <SEO title="LatestMediaNewsPage" />
+      <Typography variant="h4">{t("latest_media_news.title")}</Typography>
+      <rssapp-list id="4rfSeuaLAKEOtTc7"></rssapp-list>
```
Comment (cswbrian) [inline src/pages/latest-media-news.js]: Please wrap this by a <NewsContainer> Styled component, and add margin-top: 16px;

**Review by cswbrian — CHANGES_REQUESTED** [review]

File: src/pages/hygiene-tips.js (hunk 2 above)
Comment (johnhorsema) [reply src/pages/hygiene-tips.js]: Yes lemme revert this one.

**Review by nandiheath — APPROVED** [review]

**Pull request merged**